.papr.sh

```diff
@@ -9,7 +9,10 @@ query_github() {
     python -c "
 import sys
 import requests
-j = requests.get('https://api.github.com/repos/$repo/$resource').json()
+d = requests.get('https://api.github.com/repos/$repo/$resource')
+j = d.json()
```

Comment:
Might as well bump this down to after the status check? That way, we catch any error codes which don't even return valid json bodies.